.travis.yml

```diff
@@ -0,0 +1,61 @@
+branches:
+  only:
+    - master
+
+language: php
+
+php:
+  - '7.1'
+  - '7.2'
+  - nightly
+
+sudo: false
+dist: trusty
+
+cache:
+  pip: true
+  directories:
+    - $HOME/.composer/cache/files
+
+env:
+  global:
+    - PATH="$HOME/.local/bin:$PATH"
+    - SYMFONY_DEPRECATIONS_HELPER=weak
+    - TARGET=test
+    - UPSTREAM_URL=https://github.com/ekino/sonata.git
+
+matrix:
+  fast_finish: true
+  include:
+    - php: '7.2'
+      env: SYMFONY=3.4.*
+    - php: '7.2'
+      env: SYMFONY=4.0.*
+    - php: '7.2'
+      env: SYMFONY_DEPRECATIONS_HELPER=0
+    - php: '7.1'
+      env: SYMFONY=3.4.*
+    - php: '7.1'
+      env: SYMFONY=4.0.*
+    - php: '7.1'
+      env: SYMFONY_DEPRECATIONS_HELPER=0
+  allow_failures:
+    - php: nightly
+    - env: SYMFONY_DEPRECATIONS_HELPER=0
+
+before_install:
+  - git remote add upstream ${UPSTREAM_URL} && git fetch --all
+  - if [[ -x .travis/check_relevant_${TARGET}.sh && "$TRAVIS_PULL_REQUEST" != "false" ]]; then export RELEVANT=$(.travis/check_relevant_${TARGET}.sh); fi;
+  - if [[ ! -z ${RELEVANT} ]];then exit 0; fi;
+  - if [ -x .travis/before_install_${TARGET}.sh ]; then .travis/before_install_${TARGET}.sh; fi;
+
+install:
+  - if [ -x .travis/install_${TARGET}.sh ]; then .travis/install_${TARGET}.sh; fi;
+
+before_script:
+  - if [ -x .travis/before_script_${TARGET}.sh ]; then .travis/before_script_${TARGET}.sh; fi;
+
+script: make $TARGET
```

Comment:
reminder for adding the lint task to the process

Reply:
done @nanofelis :) thx for the reminder